recipes/ExitWaveReconstruction/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "ExitWaveReconstruction" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CDoberstein/ExitWaveReconstruction/archive/v{{ version }}.tar.gz
+  sha256: 0743821181713dea7203e5e2f6eaa3984ed923c6719a9bbbc5b005e45ae9dd49  
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - libblas
+    - fftw
+    - bzip2
+    - zlib
+    - libtiff
+    - clblas
+    - boost-cpp
+    - ocl-icd  # [linux]
+    - ocl_icd_wrapper_apple  # [osx]
+    - clhpp
+    - cimg
+    - xorg-libx11
+    - _openmp_mutex  # [linux]
```

Comment:
see the docs on openmp: https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp